sound/soc/sof/ipc4-topology.c

```diff
@@ -993,6 +993,59 @@ static int sof_ipc4_widget_assign_instance_id(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static void sof_update_hw_param(struct snd_pcm_hw_params *params,
```

Comment:
do you really need this function? Can we just set these in the function below directly?